koans/triangle.rb

```diff
@@ -15,8 +15,15 @@
 #
 def triangle(a, b, c)
   # WRITE THIS CODE
+  raise TriangleError.new, "Triangle Error" if [a, b, c].min <= 0
+  x, y, z = [a, b, c].sort
+  raise TriangleError.new, "Triangle Error" if x + y <= z
+  return :equilateral if a == b && b == c
+  return :isosceles if a == b || b == c || a == c
+  return :scalene if a != b && b != c
```

Comment:
emptyline after last return
```suggestion
  return :scalene if a != b && b != c

```